functorch/_src/make_functional.py

```diff
@@ -353,15 +359,22 @@ def compute_loss(params, x, t):
 
     If the model has any buffers, please use :func:`make_functional_with_buffers` instead.
 
+    Args:
+        model (torch.nn.Module): Input model.
+        disable_params_grad (bool): Flag to disable gradients for output parameters.
```

Comment:
Can we bikeshed this name a little more? The reason why it is confusing for me is that we're not really disabling gradients, what we are doing is turning on or off regular PyTorch Autograd tracking of the input Tensors.

Here are some potential names:
- set_requires_grad (requires_grad is the property that PyTorch autograd uses to track gradients)
- requires_grad
- enable_pytorch_autograd

Thoughts? @vfdev-5 @samdow

Reply:
Per the other discussion, I think `set_requires_grad ` and `requires_grad ` might be confusing since parameters don't _have_ to `requires_grad` and we aren't changing them if it's True

Maybe `enable_pytorch_autograd`? @vfdev-5 what do you think?

Reply:
> disable_autograd_tracking

I like this better. Eventually we'll upstream functorch into pytorch/pytorch and "enable_pytorch_autograd" will get even more confusing: "What do you mean by PyTorch autograd?"

Reply:
Same! Thanks for the suggestion @vfdev-5!